RecoPixelVertexing/PixelTriplets/plugins/CAHitQuadrupletGeneratorGPU.cc

```diff
@@ -93,15 +103,18 @@ void CAHitQuadrupletGeneratorGPU::fillDescriptions(edm::ParameterSetDescription
   desc.add<double>("hardCurvCut", 1.f / (0.35 * 87.f))->setComment("Cut on minimum curvature");
   desc.add<double>("dcaCutInnerTriplet", 0.15f)->setComment("Cut on origin radius when the inner hit is on BPix1");
   desc.add<double>("dcaCutOuterTriplet", 0.25f)->setComment("Cut on origin radius when the outer hit is on BPix1");
-  desc.add<bool>("earlyFishbone", false);
-  desc.add<bool>("lateFishbone", true);
+  desc.add<bool>("earlyFishbone", true);
```

Comment:
why the move to the `earlyFishbone` by default ?

Reply:
running the *early fishbone* now seems to be faster than the *late fishbone*,  both for triplets and quadruplets.

Reply:
now ```earlyFishbone``` runs after CA connection ONLY on connected Cells.
This makes it as fast as the lateFishbone and allows NOT to build and fit duplicates tuples.